builders/get-started/endpoints.md

```diff
@@ -59,3 +60,15 @@ To create a custom OnFinality endpoint, go to [OnFinality](https://onfinality.io
 3. Your custom API endpoint will be generated automatically
 
 ![OnFinality](/images/builders/get-started/endpoints/endpoints-2.png)
+
+### Dwellir {: #dwellir }
+
+[Dwellir](https://dwellir.com/development-nodes/){target=_blank} provides free endpoints for developers and dedicated nodes on our owned and operated bare-metal cluster. We're building custom data solutions and tooling so get in touch.
+
+To get started go to [Dwellir](https://dwellir.com/development-nodes/){target=_blank}:
+
+1. Click on **Get your Node**
+2. Select the network(s) from the list
+3. Your custom endpoint will be delivered
```

Comment:
As previously mentioned I tried both options and actually just got a very quick response back from your team about the dedicated server 🙌 🙂  -- just curious about the free tier and if a custom API key is supposed to be returned or not?

Reply:
actually just realized it might have been you haha 😅

Reply:
The process is semi-automated currently.

Reply:
So what is expected for the free tier? Still never received an API key.

Reply:
ok the email was only asking about the dedicated node, so just wanted to confirm what would happen for the free tier. thanks